File: app/src/main/java/com/karim/booksapp/data/models/pojos/SaleInfo.java

package com.karim.booksapp.data.models.pojos;

import java.util.List;
import com.squareup.moshi.Json;

public class SaleInfo {

    @Json(name = "country")
    public String country;
    @Json(name = "saleability")
    public String saleability;
    @Json(name = "isEbook")
    public Boolean isEbook;
    @Json(name = "listPrice")
    public ListPrice listPrice;
    @Json(name = "retailPrice")
    public RetailPrice retailPrice;
    @Json(name = "buyLink")
    public String buyLink;
    @Json(name = "offers")
    public List<Offer> offers = null;

}

File: app/src/main/java/com/karim/booksapp/data/models/pojos/SaleInfo.kt
package com.karim.booksapp.data.models.pojos

import com.squareup.moshi.Json

class SaleInfo {

    @Json(name = "country")
    var country: String? = null
    @Json(name = "saleability")
    var saleability: String? = null
    @Json(name = "isEbook")
    var isEbook: Boolean? = null
    @Json(name = "listPrice")
    var listPrice: ListPrice? = null
    @Json(name = "retailPrice")
    var retailPrice: RetailPrice? = null
    @Json(name = "buyLink")
    var buyLink: String? = null
    @Json(name = "offers")
    var offers: List<Offer>? = null

}
